bin/eventengine

```diff
@@ -1,16 +1,17 @@
 #!/usr/bin/env python
-#encoding: utf-8
+# encoding: utf-8
 # -*- testargs: -h -*-
 """
 The NAV eventengine deamon
 """
 
+import logging
+from nav.eventengine.daemon import main
+
```

Comment:
I believe the whole point of the original order of things here is to avoid exceptions being logged as `nav.eventengine.daemon` is loaded. This would reverse that.